metamorph/lib/support_functions.py

```diff
@@ -27,3 +27,8 @@ def setup_logging(default_path='logging.json', default_level=logging.INFO, env_k
 def storing_pretty_json(input_data, output="metamorph.json"):
```

Comment:
just a suggestion: rename to write_json_file() to follow with read_json_file() below